packages/victory-area/src/area.tsx

```diff
@@ -153,7 +153,7 @@ export const Area: React.FC<AreaProps> = (props) => {
 
   const line = renderLine
     ? React.cloneElement(
-        pathComponent!,
+        pathComponent,
```

Comment:
Got a message saying ` This assertion is unnecessary since it does not change the type of the expression.eslint[@typescript-eslint/no-unnecessary-type-assertion](https://typescript-eslint.io/rules/no-unnecessary-type-assertion)`. so I had to remove it